### ISSUE TITLE
A11y_AzureDataStudio(Schema,Dacpac,Backup)_Comparison_Screenredeader: No Status message is announced when we are clicking on "Any value" of the table.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#6797]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
Application Name: Azure Data Studio
Application Version: 1.9.0 (user setup)
Commit: 78a42e1
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.18362
Additional Details : MAS4.1.3, NVDA 2019.2, Jaws

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Schema Compare extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Schema compare from Context menu.
8. Navigate till Select Source or Select Target & hit enter.
9. Provide all the details for Source & Target.
10. Navigate till Ok button & hit enter.
11. Navigate till Compare menu from Schema compare Menu bar & hit enter.
12. Start NVDA and use up/down arrow keys to navigate to the table.
13. Check whether NVDA reads the status message or not.

**Actual:**
When we are moving to the table using up/down arrow keys, we can see value in the "Compare Details" is getting changed but for the screen reader users there is no announcement of the status message.

**Expected:**
Status message should be announced for screen reader users when the value is changed in the "Compare details window".

**Recommendations:**
Refer below link which is repository of bug fixes code snippets: https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Screenreader users will not be able to know the information about updated data.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for Reference:**
[6797_A11y_AzureDataStudio(Schema,Dacpac,Backup)_Comparison_Screenredeader.pptx](https://github.com/microsoft/azuredatastudio/files/3515043/6797_A11y_AzureDataStudio.Schema.Dacpac.Backup._Comparison_Screenredeader.pptx)

**Comment by Benjin** [comment]
Bug relates to the changing generated TSQL script displayed below the table of differences.  Question added to list regarding what sort of narration is desired, as reading out an entire TSQL script with each change may be impractical.

**Comment by Benjin** [comment]
After discussing with accessibility v-team , they agree that reading out the TSQL script doesn't make sense for this.  Waiting on SME guidance as to how (if at all) to handle these types of situations.

**Comment by hannahqin** [comment]
Following our conversation with Ryan and Alexandria, rather than change screenreader behavior, we will write documentation explaining how users can navigate to the TSQL script using a screenreader.

**Comment by udeeshagautam** [comment]
Tabbing outside the table + the documentation should be the ideal scenario mitigation here.
